rc/filetype/groovy.kak

```diff
@@ -0,0 +1,91 @@
+# Adapted from the file created by Daniel Lewan TeddyDD
+
+hook global BufCreate "(.+\.(groovy|gvy|gy|gsh))|.+[Jj]enkinsfile.*" %{
+  set-option buffer filetype groovy
+  set-option buffer comment_line '//'
+  set-option buffer comment_block_begin '/*'
+  set-option buffer comment_block_begin '*/'
```

Comment:
these 3 lines should be moved to `rc/tools/comment.kak` to avoid creating a dependency on comment.kak